src/main/java/org/opensearch/neuralsearch/ml/MLCommonsClientAccessor.java

```diff
@@ -98,13 +103,32 @@ public void inferenceSentences(
         @NonNull final String modelId,
         @NonNull final List<String> inputText,
         @NonNull final ActionListener<List<List<Float>>> listener
+    ) {
+        inferenceSentencesWithRetry(targetResponseFilters, modelId, inputText, 0, listener);
+    }
+
+    private void inferenceSentencesWithRetry(
+        @NonNull final List<String> targetResponseFilters,
+        @NonNull final String modelId,
+        @NonNull final List<String> inputText,
+        final int retryTime,
+        @NonNull final ActionListener<List<List<Float>>> listener
```

Comment:
remove @NonNull from private functions. Given that it is private function we can be sure that the inputs shouldn't be null.

Reply:
Removed.